devdoc/constbuffer_array_splitter_requirements.md

```diff
@@ -12,7 +12,7 @@ For example, it is used to upload large payloads to Blob store, split across lar
 ```c
 MOCKABLE_FUNCTION(, CONSTBUFFER_ARRAY_HANDLE, constbuffer_array_splitter_split, CONSTBUFFER_ARRAY_HANDLE, buffers, uint32_t, max_buffer_size);
 
-MOCKABLE_FUNCTION(, TARRAY(CONSTBUFFER_ARRAY_HANDLE), constbuffer_array_splitter_split_to_array_of_array, CONSTBUFFER_ARRAY_HANDLE, buffers, uint32_t, max_buffer_size);
+MOCKABLE_FUNCTION(, TARRAY(CONSTBUFFER_ARRAY_HANDLE), constbuffer_array_splitter_split_to_array_of_array, CONSTBUFFER_ARRAY_HANDLE, buffers, uint32_t, max_buffer_size, uint32_t*, payload_buffer_count);
```

Comment:
>payload_buffer_count [](http://example.com/codeflow?start=179&length=20)

I think this is actually split_buffer_arrays_count

Reply:
done